Formula/f/firefly.rb

```diff
@@ -0,0 +1,38 @@
+class Firefly < Formula
+  desc "Command-line Interface (CLI) for Hyperledger FireFly"
+  homepage "https://hyperledger.github.io/firefly/latest/"
+  url "https://github.com/hyperledger/firefly-cli/archive/refs/tags/v1.3.1.tar.gz"
+  sha256 "d2b0420b37c1ce6195e0739b2341502e65fea23c3ddd41cd55159ea237e01f23"
+  license "Apache-2.0"
+  head "https://github.com/hyperledger/firefly-cli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    # This regex is intended to match FireFly version tags (e.g. `v1.2.3`)
+    # and omit any pre-release suffixes (e.g. `v1.2.3-rc0`)
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    build_commit = if build.head?
+      `git rev-parse --short HEAD`.chomp
+    else
+      "unknown"
+    end
+
+    ldflags = %W[
+      -X github.com/hyperledger/firefly-cli/cmd.BuildDate=#{Time.now.utc.iso8601}
+      -X github.com/hyperledger/firefly-cli/cmd.BuildCommit=#{build_commit}
+      -X github.com/hyperledger/firefly-cli/cmd.BuildVersionOverride=v#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags:), "ff/main.go"
```

Comment:
Does this mean that brew will build a binary of the CLI and not use the already built and released binaries?

Reply:
Yes, https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae